### ISSUE TITLE
debian if-pre-up.d script references nonexistent /etc/iptables/general

### ISSUE THREAD

**Issue by lneva-fastly** [chef-cookbooks/iptables#103]
### Cookbook version
4.5.0, master

The line of code in question is: https://github.com/chef-cookbooks/iptables/blob/2c02699814eb9e1f16c29bcaabb1170404b07040/recipes/default.rb#L62

This should be:

```ruby
variables iptables_save_file: node['iptables']["persisted_rules_#{ipt}"]
```

Really, this block shouldn't be there at all.  It was removed in #90 because `iptables-persistent` made it obselete.  It was re-added in #87, which merged after #90, so perhaps there was some kind of mis-merge?

Doesn't seem to cause any problems, but it sure did throw me for a loop when I was debugging a separate issue.

**Comment by welcomebot** [comment]
Hey There
It looks like this is the first issue you've filed against the chef-cookbooks project. I'm here to offer you a bit of extra help to make sure we can quickly get back to you.
 Make sure you've filled out all the fields in our issue template. Make sure you've provided us with the version of chef-client you're running, your operating system and the version of the cookbook. If you're not using the most up to date version of the cookbook then please make sure to update first. Lots of things change between versions even if you're issue isn't listed in the changelog. Finally please give us a detailed description of the issue you're having. The more we know about what you're trying to do, what actually happens, and how you can reproduce the problem, the better.

If you're looking for more immediate troubleshooting help make sure to check out #general on the Chef Community Slack. There's plenty of folks there willing to lend a helping hand. Thanks for the first issue. We hope we can get back to you soon with a solution.

**Comment by Xorima** [comment]
Hey, I think @bmhughes has resolved this so closing this one, 

thanks for reporting it
